src/connectors/chillhop.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+Connector.playerSelector = '#player-controlls';
+Connector.pauseButtonSelector = '.fa-pause';
+Connector.durationSelector = '.track-length';
+Connector.artistSelector = '.jp-artists';
+
+const trackSelector = '.jp-title';
+Connector.getTrack = () => {
+	return Util.getTextFromSelectors(trackSelector);
```

Comment:
Why don't you use `trackSelector` directly?

Reply:
Fixed.